odoo/addons/base/models/image_mixin.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, api, tools
+
+
+class ImageMixin(models.AbstractModel):
+    _name = 'image.mixin'
+    _description = "Image Mixin"
+
+    # all image fields are base64 encoded and PIL-supported
+
+    # all image_raw fields are technical and should not be displayed to the user
+    image_raw_original = fields.Binary("Raw Original Image", attachment=True)
+    image_raw_original_byte_size = fields.Integer("Byte Size of the Raw Original Image", compute='_compute_image_raw', store=True)
+
+    # resized fields stored (as attachment) for performance
+    image_raw_big = fields.Binary("Raw Big-sized Image", compute='_compute_image_raw', store=True)
```

Comment:
It's not because of the `store` that I don't write it, but binary fields are default to `attachment=True` since 66f0e26f6f3f2155fcfcc55e84f2e76cbfdb2c1d

Reply:
But I guess I should add it or remove it from everywhere then, to be consistent.